test/allpoolmanager.t.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+import "../src/AllPoolManager.sol";
+
+contract AllPoolManagerTest is Test {
+    AllPoolManager public allPoolManager;
+
+    string poolName = "TestPool";
+    address token0 = address(1);
+    address token1 = address(2);
+    uint24 fee = 3000;
+    uint256 lowPrice = 100;
+    uint256 highPrice = 200;
+    uint256 amount0 = 1000;
+    uint256 amount1 = 2000;
+    uint256 liquidityAmount = 500;
+
+    function setUp() public {
+        allPoolManager = new AllPoolManager();
+        allPoolManager.createPool(poolName, token0, token1, fee, lowPrice, highPrice);
+    }
+
```

Comment:
Assert values for respective pool Contract  balances and check for values updating correctly post addition of liquidity